eoc_journal/utils.py

```diff
@@ -0,0 +1,8 @@
+"""EOC Journal XBlock - Utils"""
+
+
+def _(text):
+    """
+    Dummy `gettext` replacement to make string extraction tools scrape strings marked for translation.
+    """
+    return text
```

Comment:
Can we use the i18n runtime service to enable translations on this xblock? I did this on the skytap xblock by adding this method to the class with the XBlock mixin: https://github.com/open-craft/xblock-skytap/blob/master/xblock_skytap/skytap.py#L87-L89

Note that I did have to add a mock for the unit tests though: https://github.com/open-craft/xblock-skytap/blob/master/tests/unit/test_skytap.py#L50